tpmutil/structures.go

```diff
@@ -48,3 +48,6 @@ type responseHeader struct {
 
 // A Handle is a reference to a TPM object.
 type Handle uint32
+
+// A U16Bytes is a byte slice with 16-bit header.
+type U16Bytes []byte
```

Comment:
Why is this necessary? The encoding logic looks identical to []byte.

Reply:
The encoding is identical for []byte on TPM2.0. However tpmutil assumes that TPM1.2 uses a 32-bit header which--although true in most cases--is not the case for the TPM_SYMMETRIC_KEY that is returned from ActivateIdentity. I mentioned this in the commit comment but I'll add a comment in the file as well.